airbyte-integrations/connectors/destination-milvus/integration_tests/milvus_integration_test.py

```diff
@@ -28,26 +28,37 @@ class MilvusIntegrationTest(BaseIntegrationTest):
 
     def _init_milvus(self):
         connections.connect(alias="test_driver", uri=self.config["indexing"]["host"], token=self.config["indexing"]["auth"]["token"])
-        self._collection = Collection(self.config["indexing"]["collection"], using="test_driver")
-
-    def _clean_index(self):
-        self._init_milvus()
-        entities = self._collection.query(
-            expr="pk != 0",
-        )
-        if len(entities) > 0:
-            id_list_expr = ", ".join([str(entity["pk"]) for entity in entities])
-            self._collection.delete(expr=f"pk in [{id_list_expr}]")
+        if utility.has_collection(self.config["indexing"]["collection"], using="test_driver"):
+            utility.drop_collection(self.config["indexing"]["collection"], using="test_driver")
 
     def setUp(self):
         with open("secrets/config.json", "r") as f:
             self.config = json.loads(f.read())
-        self._clean_index()
+        self._init_milvus()
 
     def test_check_valid_config(self):
         outcome = DestinationMilvus().check(logging.getLogger("airbyte"), self.config)
         assert outcome.status == Status.SUCCEEDED
 
+    def _create_collection(self, vector_dimensions=1536):
+        pk = FieldSchema(name="pk", dtype=DataType.INT64, is_primary=True, auto_id=True)
+        vector = FieldSchema(name="vector", dtype=DataType.FLOAT_VECTOR, dim=vector_dimensions)
+        schema = CollectionSchema(fields=[pk, vector], enable_dynamic_field=True)
+        collection = Collection(name=self.config["indexing"]["collection"], schema=schema, using="test_driver")
+        collection.create_index(
+            field_name="vector", index_params={"metric_type": "L2", "index_type": "IVF_FLAT", "params": {"nlist": 1024}}
```

Comment:
Ditto comment above about explaining the constants and params used in `index_params`. 

Optionally, these calls to `create_index()` could also be refactored to a milvus-specific helper function with a docstring explaining the internals.